### PR TITLE
Add discussions file

### LINKED ISSUE #23: Add Discussion in Contribution
Currently, information regarding Discussion is missing in the documentation. It would be helpful to let contributors know regarding the ways they can contribute in Discussions.

### PR DESCRIPTION
**Pull request by Kyaa-dost**
## What changes does this PR introduce?
Adds Discussion section

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Correctly picked the forked new branch to base the change off (`main`)
- [x] Ran against a spell and grammar checker

### CODE REVIEW

File: Avoid-Roadblocks.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-* The issue you file must fulfill the requirements of [Bug report Template](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) to provide greater insight and understanding of the issue. By providing detailed information the issue will be investigated and resolved in a timely manner.
```
Comment (RosarioPulella) [inline Avoid-Roadblocks.md:1]: Maybe we should have a separate PR for updating all the WCT `master` branch references to `main`?

**Review by RosarioPulella — APPROVED** [review]
Love the new section, changing the references from `master` to `main` is good but it should have been in a separate PR. Approving anyways, I am okay with just merging this or splitting it.

File: Discussions.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[Discussions](https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions) is a great way to explore different options in the toolkit such as answering questions that are posed in the [Questions & Help]( https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions/categories/questions-help) section, contribute towards feature or ideas mentioned in the [Ideas]( https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions/categories/ideas) section, [Show and tell]( https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions/categories/show-and-tell) to explore things other community have worked on and share your own story or accomplishments, or feel free to share any general discussion in the [General](https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions/categories/general) section.
```
Comment (michael-hawker) [inline Discussions.md:1]:
```suggestion
[Discussions](https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions) are a new feature of GitHub to make it easier to engage with our community on a variety of topics that may not all be specific to 'issues' within the Toolkit itself. We prefer that folks start a discussion first unless they are confident it is an issue with something in the Toolkit's code. (This includes feature requests.)

A great way to explore different options in the toolkit is searching or answering questions that are posed in the [Questions & Help](https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions/categories/questions-help) section, contribute towards feature or ideas mentioned in the [Ideas]( https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions/categories/ideas) section, [Show and tell]( https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions/categories/show-and-tell) to explore things other community have worked on and share your own story or accomplishments, or feel free to share any general discussion in the [General](https://github.com/windows-toolkit/WindowsCommunityToolkit/discussions/categories/general) section.
```

File: Discussions.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+
+All of these are great ways to contribute and add value to the Windows Community Toolkit.
```
Comment (michael-hawker) [inline Discussions.md:3]:
```suggestion
All of these are great ways to contribute and add value to the Windows Community Toolkit and its community. 
```

File: Discussions.md (hunk 2 above)
Comment (michael-hawker) [reply Discussions.md:1]: We may also just want to make the list of sections just a bulleted list.

**Review by michael-hawker — CHANGES_REQUESTED** [review]
We probably want to make sure we call out Discussions from the Issues decision tree as well: https://github.com/windows-toolkit/WindowsCommunityToolkit/wiki/Submitting-an-Issue

File: Avoid-Roadblocks.md (hunk 1 above)
Comment (Kyaa-dost) [reply Avoid-Roadblocks.md:1]: I like the idea but we do not have many changes in the wiki in regards to Master VS Main so it should do the job for now. In the future will have to be careful though once the Wiki is more scalable.

**Review by michael-hawker — APPROVED** [review]

**Pull request merged**